### ISSUE TITLE
Issues found in #10

### ISSUE THREAD

**Issue by lim-james** [niji-co/Fusion-Web#11]
# Description

Found in [`feature/exhbition`](https://github.com/niji-co/Fusion-Web/tree/feature/exhibition) #10 
There were some issues found after the PR was accepted. @sophiewongsp can you look into them.

# Breakdown

> Most of these issues are from ESLint. It is shown in `npm start` or `yarn start`, because your scripts are not used yet. These should be resolved now, else more will build up over time. Additionally, our codebase would maintain good structure. I have attached a few extensions in [Extensions](#Extensions) section below.

## Exhbition

- [x] Missing `index.ts` file
- [x] Refactor import statements to import from index files instead
```js
// currently
import Section from "../../Section/models/SectionProps";
import Role from "../../Role/models/RoleProps";

// ideally
import { SectionProps } from "../../Section";
import { RoleProps } from "../../Role";
```

## Role 

- [x] Missing `index.ts` file
- [x] Unused Link import in `Role.tsx - line 3`. ESLint throwing the following error.
```
'Link' is defined but never used. eslint(@typescript-eslint/no-unused-vars)
```
- [x] Missing `styles.css` file. ESLint throwing the following error in `Role.tsx - line 6`.
```
Unable to resolve path to module './style.css'. eslint(import/no-unresolved)
```
- [x] Because of `{props.role}`, ESLint throwing the following error in `Role.tsx - line 10`. Refactor required.
```
Must use destructuring props assignmenteslintreact/destructuring-assignment
```
- [x] Add new line at `Role.tsx`. ESLint throwing the following error on last line (14).
```
Insert `⏎` eslint(prettier/prettier)
```
- [x] Add new line at `RoleProps.tsx`. ESLint throwing the following error on last line (14).
```
Insert `⏎` eslint(prettier/prettier)
```
- [ ] Add story for `Role.tsx` component (Documentation)

## Section 

- [x] Missing `index.ts` file
- [x] Unused Link import in `Section.tsx - line 3`. ESLint throwing the following error.
```
'Link' is defined but never used. eslint(@typescript-eslint/no-unused-vars)
```
- [x] Missing `styles.css` file. ESLint throwing the following error in `Section.tsx - line 6`.
```
Unable to resolve path to module './style.css'. eslint(import/no-unresolved)
```
- [x] Because of `{props.sectionName}` & `{props.exhibitor}`, ESLint throwing the following error in `Section.tsx - line 10`. Refactor required.
```
Must use destructuring props assignmenteslintreact/destructuring-assignment
```
- [x] Add new line at `Role.tsx`. ESLint throwing the following error on last line (14).
```
Insert `⏎` eslint(prettier/prettier)
```
- [x] Add new line at `RoleProps.tsx`. ESLint throwing the following error on last line (14).
```
Insert `⏎` eslint(prettier/prettier)
```
- [ ] Add story for `Section.tsx` component (Documentation)

# Extensions

Install the following VSCode extensions to help your with development. 
```
Name: ESLint
Id: dbaeumer.vscode-eslint
Description: Integrates ESLint JavaScript into VS Code.
Version: 2.1.20
Publisher: Dirk Baeumer
VS Marketplace Link: https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint 
```
```
Name: JavaScript and TypeScript Nightly
Id: ms-vscode.vscode-typescript-next
Description: Enables typescript@next to power VS Code's built-in JavaScript and TypeScript support
Version: 4.3.20210419
Publisher: Microsoft
VS Marketplace Link: https://marketplace.visualstudio.com/items?itemName=ms-vscode.vscode-typescript-next
```
```
Name: Prettier - Code formatter
Id: esbenp.prettier-vscode
Description: Code formatter using prettier
Version: 6.3.2
Publisher: Prettier
VS Marketplace Link: https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode
```

**Comment by lim-james** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Create Exhibition Model](https://trello.com/c/tG5xhah9/18-create-exhibition-model)

**Comment by lim-james** [comment]
Mostly resolved in #14. Missing documentation.

**Comment by lim-james** [comment]
This issue is stale. I'm closing it in the meantime. If needed, I will open a new issue with the missing items in the future.
